web/js/deck.js

```diff
@@ -597,7 +597,7 @@ function handle_quantity_change(event) {
 function update_core_sets() {
 	CardDivs = [ null, {}, {}, {} ];
 	NRDB.data.cards.find({
-		pack_code : ['core', 'core2']
+		pack_code : ['core', 'core2','sc19']
```

Comment:
Completely nitpicking, but maybe just add a space before 'sc19'.

Reply:
Yeah, I spotted this as well, thanks.
```suggestion
		pack_code : ['core', 'core2', 'sc19']
```

Reply:
Sorry for merging too fast! I'll add the fix.